_content/excerpt/mulamadhymakakarika_garfield-jay.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Mulamadhymakakarika"
```

Comment:
🤔 In this case I'm inclined to add something to the title to indicate that this isn't the entire thing... What do you think? `"[Selected Verses from] Mulamadhyamakakarika"`?